app.js

```diff
@@ -1,157 +1,209 @@
-    // Give feedback if answer is greater than 10 and less than or equal to 15.
-  }
-  else if(answerSeven > 15 && answerSeven <= 19) {
-    alert('Your close, but no cigar. Guess again but just a wee bit higher.');
-    console.log('The answer to question seven is: ' + answerSeven);
-    // Give feedback if answer is greater than 15 and less than or equal to 19.
-  }
-  else if(answerSeven == 20) {
-    alert('Wow, you are correct. Good guess!');
-    correctAnswers.push(7);
-    // Use the keyword break to stop the loop once the correct input is evaluated.
-    break;
-    // Give feedback if answer is equal to 20.
-    // Use double equals to except just the value of 20.
-  }
-  // Run the loop four times.
-} while (counter < 4);
+    if(answerSeven <= 10) {
+      alert('That\'s too low aim a little higher.');
+      console.log('The user answer to question seven is: ' + answerSeven);
+      // Give feedback if answer is less than or equal to 10.
+    }
+    else if(answerSeven > 10 && answerSeven <= 15) {
+      alert('Try again, but aim a little higher.');
+      console.log('The user answer to question seven is: ' + answerSeven);
+      // Give feedback if answer is greater than 10 and less than or equal to 15.
+    }
+    else if(answerSeven > 15 && answerSeven <= 19) {
+      alert('Your close, but no cigar. Guess again but just a wee bit higher.');
+      console.log('The user answer to question seven is: ' + answerSeven);
+      // Give feedback if answer is greater than 15 and less than or equal to 19.
+    }
+    else if(answerSeven > 20){
+      alert('Too high!');
+      console.log('The user answer to question seven is: ' + answerSeven);
+    }
+    else if(answerSeven == 20) {
+      alert('Wow, you are correct. Good guess!');
+      correctAnswers.push(7);
+      // Use the keyword break to stop the loop once the correct input is evaluated.
+      break;
+      // Give feedback if answer is equal to 20.
+      // Use double equals to except just the value of 20.
+    }
+    // Run the loop four times.
+  } while (counter < 4);
+}
+//Calling questionSeven function
+questionSeven();
 
 // Question eight.
-// Use a for loop to loop through an array six times.
-for(var index = 0; index < 6; index++) {
-  var places = ['singapore', 'maui', 'austrailia', 'dubai', 'hongkong'];
-  var answerEight = prompt('What places have I traveled to?').toLowerCase();
-  console.log('The answer to question eight is: ' + answerEight);
-  // If user's input is not outside of the array.
-  if (places.indexOf(answerEight) !== - 1) {
-    alert('Yes, I\'ve been there and would love to go back someday.');
-    correctAnswers.push(8);
-    // Use the keyword break to stop the loop.
-    break;
-  }
-  else {
-    alert('Try again, you might get it right');
+//Wrapped into a function
+function questionEight(){
+  // Use a for loop to loop through an array six times.
+  for(var index = 0; index < 6; index++) {
```

Comment:
Nice use of a loop here, though I think naming your loop's variable 'index' can actually cause some condition, since you're not iterating through an array. Index usually implies to someone reading the code that you're referring to an array's index, but that's not happening in this loop.